README.md

```diff
@@ -3,3 +3,8 @@
 | Route | Description | Parameters | Expected Output |
 |-------|-------------|------------|-----------------|
 | `POST /api/login`| logs the user in | body: `{ email: string, password: string}` | `{ token: string, refresh_token: string }` |
+| `POST /api/user?=1/create`| Create a list | body `{ title: string, description: string}` | `{ title: string, description: string}` |
+| `POST /api/sign`| Sign up user in | body: `{ firstName: string, lastName: string, email: string, password: string}` | `{ token: string, refresh_token: string }` |
```

Comment:
maybe change `/api/sign` to `/api/sign-in`

Reply:
|`POST /api/sign-in` | Sign in a user | body `{ firstName: string, lastName: string, email: string, password: string}` | `{ token: string, refresh_token: string}`|